src/components/OpenFileDialog/PaginationControls.js

```diff
@@ -0,0 +1,30 @@
+import i18n from '@dhis2/d2-i18n'
+import { Pagination } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export const PaginationControls = ({ page, pager, onPageChange }) => (
+    <Pagination
+        page={page}
+        pageCount={pager.pageCount}
+        pageSize={pager.pageSize}
+        total={pager.total}
+        onPageChange={onPageChange}
+        onPageSizeChange={Function.prototype}
```

Comment:
why `Function.prototype`?

Reply:
with `hidePageSizeSelect` used there's no need to provide a `onPageSizeChange`?

Reply:
🤔 i remember it didn't work if the callback prop was not defined.
It seems is still the case:
https://github.com/dhis2/ui/blob/master/packages/widgets/src/Pagination/Pagination.js#L137
and there is no `default` for it.
It looks like a 🐛

Reply:
oh ok, if that's the case, would you mind raising a ticket for `ui` and add a comment with the ticket on the `onPageSizeChange` line? Feels like we'll be scratching our heads otherwise in the future 🙂

Reply:
Opened https://github.com/dhis2/notes/discussions/295